.github/workflows/maven-package.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '21-ea' ]
+        java: [ '17', '21' ]
```

Comment:
This is not what we discussed. It simply runs two parallel builds, one using JDK 17 and one using JDK 21. For a MR, we just need a single build on JDK 21 but with two compilation targets, 17 and 21.

Reply:
We currently have no reason for a MR JAR. We can consider adding it back, but without the need for a `META-INF/versions/21` it makes no sense.

Reply:
The MR was the last that you and me agreed upon, but indeed, now that Arjan vetoed against the MR, my comment is obsolete.